doc/lxc.container.conf.sgml.in

```diff
@@ -510,6 +510,41 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
               mode is possible for one physical interface.
             </para>
 
+            <para>
+              <option>ipvlan:</option> an ipvlan interface is linked
+              with the interface specified by
+              the <option>lxc.net.[i].link</option> and assigned to
+              the container.
+              <option>lxc.net.[i].ipvlan.mode</option> specifies the
+              mode the ipvlan will use to communicate between
+              different ipvlan on the same upper device. The accepted
+              modes are <option>l3</option>, <option>l3s</option> and
+              <option>l2</option>. It defaults to <option>l3</option> mode.
+              In <option>l3</option> mode TX processing up to L3 happens on the stack instance
+              attached to the slave device and packets are switched to the stack instance of the
+              master device for the L2 processing and routing from that instance will be
+              used before packets are queued on the outbound device. In this mode the slaves
+              will not receive nor can send multicast / broadcast traffic.
+              In <option>l3s</option> mode TX processing is very similar to the L3 mode except that
+              iptables (conn-tracking) works in this mode and hence it is L3-symmetric (L3s).
+              This will have slightly less performance but that shouldn't matter since you are
+              choosing this mode over plain-L3 mode to make conn-tracking work.
+              In <option>l2</option> mode TX processing happens on the stack instance attached to
+              the slave device and packets are switched and queued to the master device to send
+              out. In this mode the slaves will RX/TX multicast and broadcast (if applicable) as well.
+              <option>lxc.net.[i].ipvlan.flags</option> specifies a mode flag to influence
```

Comment:
@brauner @stgraber the new `ipvlan.flags` property closely mirrors the naming used in the kernel, however the use of the term "flags" suggests it could be expanded later on and that a device could take multiple flags. I would like your opinion on whether for LXC's purposes we rename this property to reflect the effect these 3 specific flags have on functionality today, namely they affect varying levels of container isolation. I was thinking perhaps calling this `ipvlan.isolation` instead.

Reply:
Indeed isolation may be a better name for this an if they do add a different class of flags, we could then add another key and generate the right set of flags internally.

Reply:
@stgraber great, I'll make those changes